core/src/executor/select/mod.rs

```diff
@@ -124,8 +124,33 @@ pub async fn select_with_labels<'a>(
         order_by,
     } = match &query.body {
         SetExpr::Select(statement) => statement.as_ref(),
-        _ => {
-            return Err(SelectError::Unreachable.into());
+        SetExpr::Values(Values(values_list)) => {
+            let limit = Limit::new(query.limit.as_ref(), query.offset.as_ref())?;
+            let labels = values_list
+                .iter()
+                .enumerate()
+                .map(|(i, _)| format!("column{}", i + 1))
+                .collect::<Vec<_>>();
+            let first_len = values_list[0].len();
```

Comment:
Will there be any possibility when users will accidentally call values list WITHOUT any actual list items?

e.g. 
```sql
# Calling without any list
gluesql> VALUES;
gluesql> ??
```

If this happens, maybe this `values_list[0].len()` will omit error or warning due to indexing empty list.

Reply:
Thank you for pointing out !
Fortunatly, those kinda SQLs are invalid so that it gets ParseError before execute the line !

Reply:
Thats awesome! Thanks

Reply:
Not necessary work for now, but we may need to use a new data structure which can describe non-empty-vector.
maybe you can add `NonEmptyVec` to `/utils`?
Then, `NonEmptyVec` can become a fundamental solution for this kind of situations.